From fee054dfaf9cf4dc389c8b35ee28c45c2dc63952 Mon Sep 17 00:00:00 2001
From: Moritz Fischer <moritz.fischer@ettus.com>
Date: Wed, 23 Sep 2015 15:16:10 -0700
Subject: [PATCH 3/3] uhd: C API wrapper, fix commit 30f87afcba71a07

The uhd/usrp_clock/usrp_clock.h doesn't get properly installed,
which isn't a big deal if examples are built in tree, but if
someone attempts to build a C application against libuhd,
uhd.h pulls in <uhd/usrp/usrp_clock.h> and fails as follows:

fatal error: uhd/usrp_clock/usrp_clock.h: No such file or directory

Signed-off-by: Moritz Fischer <moritz.fischer@ettus.com>
---
 host/include/uhd/usrp_clock/CMakeLists.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/host/include/uhd/usrp_clock/CMakeLists.txt b/host/include/uhd/usrp_clock/CMakeLists.txt
index 518cb77..c8c0e43 100644
--- a/host/include/uhd/usrp_clock/CMakeLists.txt
+++ b/host/include/uhd/usrp_clock/CMakeLists.txt
@@ -24,6 +24,8 @@ UHD_INSTALL(FILES
 
 IF(ENABLE_C_API)
     UHD_INSTALL(FILES
-        usrp_clock.h
+    usrp_clock.h
+    DESTINATION ${INCLUDE_DIR}/uhd/usrp_clock
+    COMPONENT headers
     )
 ENDIF(ENABLE_C_API)
-- 
2.1.4

